public/video-ui/src/components/VideoUsages/VideoUsages.js

```diff
@@ -67,42 +61,50 @@ export default class VideoUsages extends React.Component {
           >
             <ViewerIcon />
           </a>
+
+          {state === ContentApi.published
+            ? <a className="usage--platform-link"
+                 href={websiteLink}
+                 title="Open on theguardian.com"
+                 target="_blank"
+                 rel="noopener noreferrer">
+              <FrontendIcon />
+            </a>
+            : ''}
         </div>
       </li>
     );
   };
 
   renderUsages() {
-    return (
-      <ul className="detail__list">
-        {this.props.usages.map(this.renderUsage)}
-      </ul>
-    );
-  }
+    return Object.keys(this.props.usages).map(state => {
+      const totalUsages = this.props.usages[state].video.length + this.props.usages[state].other.length;
 
-  render() {
-    if (!this.props.usages) {
-      return <div className="baseline-margin">Fetching Usages...</div>;
-    }
-
-    if (this.props.usages.length === 0) {
-      return (
-        <div>
-          <div className="baseline-margin">No usages found</div>
-        </div>
-      );
-    } else {
       return (
-        <div>
-          {this.renderUsages()}
+        <div key={`${state}-usages`}>
+          <h3>{state.charAt(0).toUpperCase() + state.slice(1)}</h3>
+          {totalUsages === 0
+            ? <div className="baseline-margin">{`No ${state} usages found`}</div>
+            : <ul className="detail__list">
+                {this.props.usages[state].video.map(usage => this.renderUsage({usage, state}))}
+                {this.props.usages[state].other.map(usage => this.renderUsage({usage, state}))}
+              </ul>}
         </div>
       );
-    }
+    });
+  }
+
+  render() {
+    return (
+      <div className="form__group">
+        {this.renderUsages()}
+      </div>
+    );
   }
 }
 
 VideoUsages.propTypes = {
-  usages: PropTypes.array.isRequired,
+  // usages: PropTypes.object.isRequired,
```

Comment:
oh yeah! I was having some problems with this... will check if I accidentally fixed it :)